Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/BindingContract.cs

```diff
@@ -42,12 +44,12 @@ public sealed class BindingContract : Entity
         DateTimeOffset bindingFrom,
         DateTimeOffset expiringAt) => new(id, customerId, duration, bindingFrom, expiringAt);
 
-    public Annex AddAnnex(DateTimeOffset validFrom, DateTimeOffset now)
+    public void AddAnnex(DateTimeOffset validFrom, DateTimeOffset now)
```

Comment:
What you think to rename this method to "Attach"?

I've checked some business conversations in the Internet and I think is more appropriate.

```suggestion
    public void AttachAnnex(DateTimeOffset validFrom, DateTimeOffset now)
```

Reply:
Great input, love it!